packages/core/src/service/plugin/index.js

```diff
@@ -162,6 +176,51 @@ export default class PluginService {
     ...this.plugins.layout.map(({ name }: Plugin) => name)
   ]
 
+  migratePluginState = (state: any, plugin: Plugin, dataVersion: string): ?Object => {
+    if (!plugin || !dataVersion || semver.valid(dataVersion) === null) {
+      return state
+    }
+    let currentDataVersion = dataVersion
+    let migrations = plugin.migrations ? plugin.migrations : []
+    while (true) {
+      const migration = migrations.find(m => semver.satisfies(currentDataVersion, m.fromVersionRange))
+      migrations = migrations.filter(m => !semver.satisfies(currentDataVersion, m.fromVersionRange))
```

Comment:
This does not seem to be used in the rest of the code @PeterKottas

Reply:
I think the code selection went bananas. Cause what I see mentioned is the migratePluginState function and that is definitely used. That's what migrate the state.

Reply:
My bad, I read the code wrong

Reply:
Ah cool, np